### DISCUSSION TITLE
I would like to know what open source projects are generally used for, and are there many people who can accept them?

### DISCUSSION THREAD

**Discussion by xuuhu** [community/community#65257] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to write an open source project, which depends on many projects, so that more people can participate. Do you have any suggestions?

**Comment by leikang123** [comment]
You can write about a basic project, for example, editors, third-party libraries, middleware, plug-ins for various programming languages, etc., you can check which dependent projects on Google, as for the participants, you have to Publish the advantages of this project in the place where it is released, so that more people can participate

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
